### DISCUSSION TITLE
创建JDBC-catalog，如果密码错误后是否有重试机制，会重试几次？

### DISCUSSION THREAD

**Discussion by doublewaters** [apache/doris#40114] [A - General / Q&A]
创建JDBC-catalog，如果密码错误后是否有重试机制，会重试几次？

**Comment by ixzc** [comment]
建议英文提问，如使用中文提问，可以去ask.selectdb.com中文技术论坛。

**Comment by zy-kkk** [comment]
When creating a Jdbc Catalog, a connection check is performed by default. This is performed on a random FE and BE. If the password is incorrect, the creation of the Catalog will be prohibited. This avoids frequent attempts to use the wrong password on the source library. If the password of the source library is changed midway, it is recommended to stop using or delete the current Catalog first, and then change the password of the Catalog or create the Catalog with the new password after changing the source password. This is to avoid the connection pool constantly trying to connect with the old password to keep the connection alive during the password change process.
Note that the connection test function for creating a Catalog is only on the newer release branch.
